mslib/mscolab/conf.py

```diff
@@ -155,6 +158,7 @@ class setup_saml2_backend:
                                            Ignore this warning when you initializeing metadata.")
 
                         localhost_test_idp = SPConfig().load(yaml_data["config"]["localhost_test_idp"])
+                        localhost_test_idp.verify_ssl_cert = mscolab_settings.VERIFY_SSL_CERT
```

Comment:
I am not sure about the possibility of adding a single value since we are passing the SAML2Client through the dict, and `idp_identity_name`  also be defined by the user when configuring multiple IdPs.. hint added.